packages/server/src/api/contributionsApi.ts

```diff
@@ -24,25 +11,15 @@ export const variantHasArticleCountCopy = (variant: EpicVariant): boolean => {
     );
 };
 
-export const fetchConfiguredEpicTests = async (testList: EpicTestList): Promise<EpicTest[]> => {
-    const file = EPIC_TEST_LIST_FILE_LOOKUP[testList];
-    const key = `epic/${isProd ? 'PROD' : 'CODE'}/${file}`;
-
-    return fetchS3Data('gu-contributions-public', key)
-        .then(JSON.parse)
-        .then(json => {
-            const { tests } = json;
-            if (Array.isArray(tests)) {
-                return tests.map((test: EpicTest) => {
-                    const hasArticleCountInCopy = test.variants.some(variantHasArticleCountCopy);
-
-                    return {
-                        ...test,
-                        hasArticleCountInCopy,
-                    };
-                });
-            }
+export const fetchConfiguredEpicTests = async (channel: ChannelTypes): Promise<EpicTest[]> => {
```

Comment:
let's rename/move this file to `packages/server/src/tests/epic/epicTests.ts`

Reply:
👍